l10n_cl_electronic_invoicing/readme/CONTRIBUTORS.rst

```diff
@@ -2,3 +2,4 @@
 * Nelson Ramirez
 * Carlos Toledo
 * Carlos Lopez
+* Flectra Chile
```

Comment:
@flectra-chile We will need firstname and lastname of individuals here with email address if possible. Thank you.

Reply:
@flectra-chile the name itself is confrontative and conflictive in my opinion

Reply:
> @flectra-chile We will need firstname and lastname of individuals here with email address if possible. Thank you.

@max3903  Flectra Chile is SpA, individual contributor is Francisco Lorca <florca1979@gmail.com>

I Update PR for avoid confusion